docs/graphcache/gotchas.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Gotchas
+order: 5
+---
+
+# Gotchas
```

Comment:
Spewing some suggestions for titles

 - Tips & tricks 
 - Under the hood
 - Additional info
 - Things to note
 - Special updates
 - Functional nuances

Reply:
I think “Under the good” gets the gist right, as it indicates “hidden features” that work in the background, but sounds a little slangy 😄

Reply:
Let's go with "Under the hood" for now if we can't come up with anything more apt yet?